### DISCUSSION TITLE
一个问题(仅仅是好奇(

### DISCUSSION THREAD

**Discussion by LTDove** [Meloong-Git/PCL#1019] [提问] [answered]
如果公测版用户提交了反馈并完成以后会考虑发些小奖品吗(?
如公测专属的主题啥的(?

**Comment by miner233** [comment]
不是已经有了吧，活跃橙？
PCL2 Repo 发 issues 或者在帮助 Repo 里面发 Pull requests 龙猫合并到主分支后就可以找龙猫要解锁码了...
也可以通过参加内群活动获得
赞助就发个主题...不也已经有了吗？秋仪金，但是得达到 23.33 元才能有...

**Reply by WForst-Breeze** [reply]
注意审题（）**公测版**

**Reply by LTDove** [reply]
公测版.....

**Comment by WForst-Breeze** [comment, marked as answer]
~好家伙这个问题居然还没有被采纳的回答~
概率不太大个人认为，貌似能发的奖品现在除了主题的确我暂时也想不出别的了（）实体奖励那就更不现实了。
但是主题更换**目前是** Snapshot 版本才可以使用，如果有了什么公测**专属**主题，可能就不大妥了QwQ……尽管龙猫**曾计划过**重置所有主题并为公测版支持部分主题的更换，但是就是说毕竟……优先级很低，可能一时半会是不会做出来的。综上，我感觉可能性不太大
具体的决定还是以龙猫为准~
